sdk/core/core-rest-pipeline/src/util/helpers.ts

```diff
@@ -1,22 +1,71 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
+import { AbortError, AbortSignalLike } from "@azure/abort-controller";
+
 /**
  * A constant that indicates whether the environment the code is running is Node.JS.
  * @internal
  */
 export const isNode =
   typeof process !== "undefined" && Boolean(process.version) && Boolean(process.versions?.node);
 
+const StandardAbortMessage = "The operation was aborted.";
+
 /**
- * A wrapper for setTimeout that resolves a promise after t milliseconds.
- * @internal
- * @param t - The number of milliseconds to be delayed.
+ * A wrapper for setTimeout that resolves a promise after delayInMs milliseconds.
+ * @param delayInMs - The number of milliseconds to be delayed.
  * @param value - The value to be resolved with after a timeout of t milliseconds.
- * @returns Resolved promise
+ * @param options - The options for delay - currently abort options
+ *   @param abortSignal - The abortSignal associated with containing operation.
```

Comment:
Does tsdoc/jsdoc work in this way for nested properties?

Reply:
Hmmm, doesn't look good.

Updated to 
![image](https://user-images.githubusercontent.com/10452642/157792074-e56fd22e-8a09-4046-b21d-4cded920ea2a.png)
 
Ok now?